packages/react/src/modal/ModalDialog.tsx

```diff
@@ -57,15 +56,11 @@ export const ModalDialog = ({
 					</Box>
 				) : null}
 				<Button
-					variant="tertiary"
+					variant="text"
 					aria-label="Close modal"
 					onClick={onDismiss}
 					iconAfter={CloseIcon}
-					css={{
-						position: 'absolute',
-						top: mapSpacing(0.5),
-						right: mapSpacing(0.5),
-					}}
+					css={{ order: -1, alignSelf: 'flex-end' }}
```

Comment:
These button changes make it A LOT easier to position the close button in the top right of the modal, while still keeping the DOM order as is.

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/order